src/entrypoint.ts

```diff
@@ -1,12 +1,12 @@
 import {
-  IssuesAddLabelsParams,
-  IssuesAddLabelsResponseItem,
-  IssuesListLabelsOnIssueParams,
-  IssuesListLabelsOnIssueResponse,
-  PullsListFilesParams,
-  PullsListFilesResponse,
-  PullsListFilesResponseItem,
-  Response
+    IssuesAddLabelsParams,
```

Comment:
please check your indent to make sure it is `2` for typescript and format / commit so we get only the changes I did in this file ;-)